AndroidManifest.xml

```diff
@@ -23,7 +23,6 @@
     android:versionName="${versionName}">
 
     <uses-sdk
-        android:minSdkVersion="14"
         android:targetSdkVersion="26"
```

Comment:
Please move `targetSdkVersion` to build.gradle as well, as we did in https://github.com/owncloud/android-library/pull/198/